### ISSUE TITLE
[Next 9] build fail

### ISSUE THREAD

**Issue by Drapegnik** [zeit/next.js#8137]
# Bug report

## Describe the bug
After update to `next@9` `next build` failed with following error message:

```bash
> cross-env NODE_ENV=production next build

Creating an optimized production build ...


Found experimental config:
Experimental features can change at anytime and aren't officially supported (use at your own risk).


> Using external babel configuration
> Location: "/Users/drapegnik/projects/babajka/frontend/.babelrc"
Compiled successfully.

> Build error occurred
{ UNKNOWN_VALUE: Unknown value: build
    at commandLineArgs (/Users/drapegnik/projects/babajka/frontend/node_modules/command-line-args/index.js:54:21)
    at module.exports.tIZw.module.exports (/Users/drapegnik/projects/babajka/frontend/.next/server/static/lUq_Ys04r11OaxeXMvfSt/pages/_app.js:5274:10)
    at Object.gLbf (/Users/drapegnik/projects/babajka/frontend/.next/server/static/lUq_Ys04r11OaxeXMvfSt/pages/_app.js:4311:12)
    at __webpack_require__ (/Users/drapegnik/projects/babajka/frontend/.next/server/static/lUq_Ys04r11OaxeXMvfSt/pages/_app.js:23:31)
    at Object.ZPNs (/Users/drapegnik/projects/babajka/frontend/.next/server/static/lUq_Ys04r11OaxeXMvfSt/pages/_app.js:3763:76)
    at __webpack_require__ (/Users/drapegnik/projects/babajka/frontend/.next/server/static/lUq_Ys04r11OaxeXMvfSt/pages/_app.js:23:31)
    at Object.dr+i (/Users/drapegnik/projects/babajka/frontend/.next/server/static/lUq_Ys04r11OaxeXMvfSt/pages/_app.js:4186:72)
    at __webpack_require__ (/Users/drapegnik/projects/babajka/frontend/.next/server/static/lUq_Ys04r11OaxeXMvfSt/pages/_app.js:23:31)
    at Module.1TCz (/Users/drapegnik/projects/babajka/frontend/.next/server/static/lUq_Ys04r11OaxeXMvfSt/pages/_app.js:866:12)
    at __webpack_require__ (/Users/drapegnik/projects/babajka/frontend/.next/server/static/lUq_Ys04r11OaxeXMvfSt/pages/_app.js:23:31) name: 'UNKNOWN_VALUE', value: 'bu
ild' }
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! babajka-frontend@1.0.1 build: `cross-env NODE_ENV=production next build`
npm ERR! Exit status 1
```

## To Reproduce

1. Go to https://github.com/babajka/babajka-frontend/tree/next-9
2. `npm i`
3. `npm run build`

## Screenshots

<img width="1121" alt="Screen Shot 2019-07-27 at 18 08 33" src="https://user-images.githubusercontent.com/11758660/61996199-60fa9d80-b09a-11e9-8bcb-4abacd4b6ca8.png">

## System information

- OS: `macOS`
- Version of Next.js: `9.0.2`, the same with `next@canary`

## Additional context

> [`next.config.js`](https://github.com/babajka/babajka-frontend/blob/next-9/next.config.js#L20)

- `node@10.14.1`
- `npm@6.10.0`
- `next-compose-plugins@2.1.1`
- `next-fonts@0.17`
- `@zeit/next-bundle-analyzer@0.1.2`
- `@zeit/next-css@1.0.1`
- `@zeit/next-sass@1.0.1`

**Comment by mkatrenik** [comment]
I see similar bug, but only on 9.0.3 (9.0.2 is ok)

Suddenly, importing one variable from one file, fails the build, everything else still works though...
One observation - in development mode, after page refresh, i see same error on the screen for a brief moment, but immediatly it recompiles a then everything works...

```
Build error occurred
{ ReferenceError: upAxios is not defined
    at Object.k (/Users/martin/Dev/brackets/up/.next/server/static/gxrlDiFKyJr1PLOjnnc_F/pages/apply-promo.js:10030:93)
    at Object.CR6n (/Users/martin/Dev/brackets/up/.next/server/static/gxrlDiFKyJr1PLOjnnc_F/pages/apply-promo.js:4175:111)
    at __webpack_require__ (/Users/martin/Dev/brackets/up/.next/server/static/gxrlDiFKyJr1PLOjnnc_F/pages/apply-promo.js:29:31)
    at Object.jU8A (/Users/martin/Dev/brackets/up/.next/server/static/gxrlDiFKyJr1PLOjnnc_F/pages/apply-promo.js:14662:62)
    at __webpack_require__ (/Users/martin/Dev/brackets/up/.next/server/static/gxrlDiFKyJr1PLOjnnc_F/pages/apply-promo.js:29:31)
    at Object.LeJ0 (/Users/martin/Dev/brackets/up/.next/server/static/gxrlDiFKyJr1PLOjnnc_F/pages/apply-promo.js:10036:83)
    at __webpack_require__ (/Users/martin/Dev/brackets/up/.next/server/static/gxrlDiFKyJr1PLOjnnc_F/pages/apply-promo.js:29:31)
    at Module.ahZQ (/Users/martin/Dev/brackets/up/.next/server/static/gxrlDiFKyJr1PLOjnnc_F/pages/apply-promo.js:12702:75)
    at __webpack_require__ (/Users/martin/Dev/brackets/up/.next/server/static/gxrlDiFKyJr1PLOjnnc_F/pages/apply-promo.js:29:31)
    at Object.2 (/Users/martin/Dev/brackets/up/.next/server/static/gxrlDiFKyJr1PLOjnnc_F/pages/apply-promo.js:1063:18) type: 'ReferenceError', '$error': '$error' }
```

**Comment by Drapegnik** [comment]
Seems that this problem occurred because I had [`webpack`](https://github.com/babajka/babajka-frontend/blob/next-9/package.json#L82) in my `package.json`

**Comment by Drapegnik** [comment]
Nope, still here with `9.0.7` :(

**Comment by Drapegnik** [comment]
Fixed with `npm i -S command-line-args@4.0.7`

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
